code/game/machinery/bots/medbot.dm

```diff
@@ -309,7 +310,7 @@
 			stunned = 0
 		return
 
-	if(frustration > 8)
+	if(frustration >= 8)
```

Comment:
It would result in it basically being > 7. It's completely the same. I just changed the medbot to have the same range as the secbot. If you want I can revert, but this makes the medbot more determined by 1 tile